lib/pages/chat/chat_pinned_events/pinned_messages_screen.dart

```diff
@@ -156,71 +157,9 @@ class PinnedMessagesScreen extends StatelessWidget {
             ),
           ),
-          ValueListenableBuilder<List<Event>>(
-            valueListenable: controller.selectedEvents,
-            builder: (context, selectedEvents, child) {
-              if (selectedEvents.isEmpty) return child!;
-
-              return Padding(
-                padding: PinnedMessagesStyle.actionBarParentPadding,
-                child: Material(
-                  elevation: 1,
-                  borderRadius: BorderRadius.circular(
-                    PinnedMessagesStyle.actionBarBorderRadius,
-                  ),
-                  child: Container(
-                    height: PinnedMessagesStyle.unpinButtonHeight,
-                    width: PinnedMessagesStyle.unpinButtonWidth,
-                    padding: PinnedMessagesStyle.actionBarPadding,
-                    decoration: BoxDecoration(
-                      color: Theme.of(context).colorScheme.surfaceVariant,
-                      borderRadius: BorderRadius.circular(
-                        PinnedMessagesStyle.actionBarBorderRadius,
-                      ),
-                    ),
-                    child: Row(
-                      children: [
-                        IconButton(
-                          onPressed: controller.closeSelectionMode,
-                          icon: Icon(
-                            Icons.close,
-                            color:
-                                Theme.of(context).colorScheme.onSurfaceVariant,
-                            size: 20,
-                          ),
-                        ),
-                        Text(
-                          L10n.of(context)!
-                              .messageSelected(selectedEvents.length),
-                          style:
-                              Theme.of(context).textTheme.labelLarge?.copyWith(
-                                    color: Theme.of(context)
-                                        .colorScheme
-                                        .onSurfaceVariant,
-                                  ),
-                        ),
-                        const Spacer(),
-                        TextButton.icon(
-                          onPressed: () => controller.unpinSelectedEvents(),
-                          icon: PinnedMessagesStyle.unpinIcon(),
-                          label: Text(
-                            L10n.of(context)!.unpin,
-                            style: Theme.of(context)
-                                .textTheme
-                                .labelLarge
-                                ?.copyWith(
-                                  color: Theme.of(context).colorScheme.primary,
-                                ),
-                          ),
-                        ),
-                      ],
-                    ),
-                  ),
-                ),
-              );
-            },
-            child: const SizedBox.shrink(),
-          ),
+          responsiveUtils.isMobileOrTablet(context)
```

Comment:
we should use PlatformInfo.isMobile ? because in web tablet mode, mobile mode it's not correct
<img width="1179" alt="Screenshot 2024-01-31 at 09 30 44" src="https://github.com/linagora/twake-on-matrix/assets/43041967/04a83b9b-d763-4377-8d93-70910b388006">

Reply:
PlatformInfo uses OS but I agree with responsiveUtils.isMobile